tripleo/README.md

```diff
@@ -97,11 +97,31 @@ pass that during overcloud deployment. The custom ceph role looks something like
 The info regarding creating a custom role is found in https://docs.openstack.org/tripleo-docs/latest/install/advanced_deployment/custom_roles.html
 
 So when composable services MetricsQdr and Collectd is enabled along with internal_tls then overcloud should be deployed as 
-    
+
     openstack overcloud deploy \
     ...
     ...
     -r ~/roles_data.yaml \
     ...
     ...
-  
+
+
+# Deploying with distributed compute nodes
+
+If the compute nodes are configured as edge nodes, and are on a separate network to the controller, a few changes are need to the documentation found in https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/spine_leaf_networking/
```

Comment:
Sorry, this sentence is a bit confusing and I'm not quite sure what is intended to be stated here :)

I think it might need to be something like:

> If the compute nodes are operating in edge mode and are on a separate network from the typical `InteralApi` network used by the controllers, a few changes are necessary beyond the standard setup as documented at https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/spine_leaf_networking/

Reply:
It's really an issue with spine & leaf, not specifically DCN, so it's probably best to keep this generic.

In fact spine & leaf is a network topology used within a single DC, so couldn't be less Edge related, but the term has been (ab)used to refer to deploying with L3 routed networks in tripleo.

Also it's not just compute nodes that could be placed on different subnets.

So maybe something like:

> # Deploying with routed L3 networks
> 
> If nodes are on a separate network from the typical `InteralApi` network used by the controllers, which is typical in a Spine & Leaf or a Distributed Compute Node topology, then a few changes are necessary...

Reply:
Hey @olliewalsh thanks for the feedback!

We've placed this into the https://github.com/redhat-service-assurance/telemetry-framework repo (which is where the documentation lives for Service Assurance). We'll address this and make sure we're clear that this is a spine+leaf configuration issue, and not specifically DCN.

Thanks for the feedback!